operations/research-and-development/product/product-ownership-areas.md

```diff
@@ -0,0 +1,13 @@
+# Product Management Areas of Ownership
+
+| Product Area | Example Features| PM Owner |
+| :--- | :--- | :--- |
+| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions| Adam Clarkson |
+| Apps & Integrations | Plugins, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
+| Customer Journey (On-Prem) | Trials, Purchases/Customer Portal, In-product notices, Admin Adivsor, Licenses | Jason Blais |
```

Comment:
Propose we combine customer journey on-prem + cloud into one row and note ownership breakdown in PM Owner column

Reply:
```suggestion
| Customer Journey | Trials, Billing, Purchases/Customer Portal, In-product notices, Admin Advisor, Licenses | Jason Blais (Self-Managed), Katie Wiersgalla (Cloud) |
```